.github/workflows/draft-release.yml

```diff
@@ -8,14 +8,14 @@ jobs:
     steps:
       - name: Get release version
         id: releaseVersion
-        uses: actions/github-script@7a5c598405937d486b0331594b5da2b14db670da # 6.1.0
+        uses: actions/github-script@v6
```

Comment:
Should we pin to the latest version? I believe the intention of the previous approach was exactly to not automatically update to whatever the action author uploads, but instead to keep on a specific version until we decide to move.

Reply:
If github has been a good citizen regarding major versions on their GH actions, I don't see why not trust their semver.

Reply:
I volunteer for this project to be a guinea pig and replace all git-sha with versions - if it breaks, this is not going to be a major issue, this library is maintained by a single team/person